pkg/interop/native/management/management.go

```diff
@@ -35,6 +42,19 @@ func GetContract(addr interop.Hash160) *Contract {
 	return neogointernal.CallWithToken(Hash, "getContract", int(contract.ReadStates), addr).(*Contract)
 }
 
+// GetContractByID represents `getContractById` method of the Management native contract.
+func GetContractByID(id int) *Contract {
+	return neogointernal.CallWithToken(Hash, "getContractById", int(contract.ReadStates), id).(*Contract)
+}
+
+// GetContractHashes represents `getContractHashes` method of the Management
+// native contract. It returns an Iterator over the list of non-native contract
+// hashes Each iterator value can be cast to IdHash. Use [iterator] interop
+// package to work with the returned Iterator.
+func GetContractHashes() iterator.Iterator {
```

Comment:
Can we add tests for these interops using neotest framework like it was done for `address.ToHassh160` and `address.FromHash160`?

Reply:
We usually test these thin wrappers in a bit different fashion, this test is added. Doing complete tests is preferable of course, but that's a separate task I think.

Reply:
See https://github.com/nspcc-dev/neo-go/issues/2709, let it be there, it may be helpful someday.